drivers/spi/Kconfig.mcux_flexcomm

```diff
@@ -1,9 +1,18 @@
 # Copyright (c) 2016, Freescale Semiconductor, Inc.
-# Copyright (c) 2017,2019, NXP
+# Copyright (c) 2017-2020, NXP
 # SPDX-License-Identifier: Apache-2.0
 
 config SPI_MCUX_FLEXCOMM
 	bool "MCUX FLEXCOMM SPI driver"
 	depends on HAS_MCUX_FLEXCOMM
 	help
 	  Enable support for mcux flexcomm spi driver.
+
+if SPI_MCUX_FLEXCOMM
+config SPI_MCUX_FLEXCOMM_DMA
```

Comment:
Should this be a config option, or just based on the existence or not of the dma property in the DTS?

Reply:
It gives user the option to enable/disable DMA based transfers through the config instead of having to delete the DMA property in the DTS.